src/hopebase/views/api.py

```diff
@@ -6,12 +6,15 @@
 
 from hopebase import serializers, permissions
 from hopebase.models import UserProfile
+from hopebase.pagination import StandardResultsSetPagination
 
 
 class UserView(generics.RetrieveAPIView):
     permission_classes = [getattr(permissions, p) for p in settings.ETHNICITY_PERMS]
     required_scopes = ['account']
     model = User
+    pagination_class = StandardResultsSetPagination
```

Comment:
Setting REST_FRAMEWORK[DEFAULT_PAGINATION_CLASS]='hopebase.pagination.StandardResultsSetPagination' will make pagination available for all queries in api, which will be very useful for android app. And no need to set it for every view.